sphinx/source/docs/user_guide/concepts.rst

```diff
@@ -135,6 +135,49 @@ Another way to generate various kinds of output is by using Bokeh's powerful
 
 .. _userguide_interfaces:
 
+Bokeh settings
+--------------
+
+There are various global settings that influence how Bokeh operates.
+
+There are several methods you can use to change Bokeh's configuration. Directly
+in the Python code, in a YAML configuration file, or with environment variables,
+for example. A full list of all available settings and how to change them is
+available at :ref:`bokeh.settings`.
+
+Some of most commonly used settings are:
+
+* ``browser`` (environment variable ``BOKEH_BROWSER``): Set this configuration
+  value to the browser you want Bokeh to use (for example when calling
+  :func:`~bokeh.io.show`). Valid values are any of the predefined browser names
+  of the Python webbrowser_ module. For example: ``chromium-browser`` or
+  ``windows-default``. You can also set this variable to the full path of your
+  browser. For example:
+
+  .. code-block:: PowerShell
+
+      $env:BOKEH_BROWSER="C:/Program\ Files/Google/Chrome/Application/chrome.exe %s &"
+
+* ``legacy`` (environment variable ``BOKEH_LEGACY``): Set this configuration
+  value to ``True`` to enable support for legacy browsers (IE and phantomjs).
+
+* ``resources`` (environment variable ``BOKEH_RESOURCES``): To display
+  interactive visualizations in a browser, Bokeh needs to load :term:`BokehJS`.
+  Set this configuration value to define where to load BokehJS from. For
+  example:
+
+  * ``cdn`` to load BokehJS from Bokeh's Content Delivery Network (CDN)
+  * ``server`` to load from a Bokeh server
+  * ``relative`` to load a local version relative to the given directory.
+
+  All available options are listed at :class:`~bokeh.resources.Resources`.
+
+  You can combine some of the values for this variable with other
+  configuration values, such as ``cdn_version`` (``BOKEH_CDN_VERSION``) and
+  ``rootdir`` (``BOKEH_ROOTDIR``). See :ref:`bokeh.settings` for details.
+
+.. _webbrowser: https://docs.python.org/3/library/webbrowser.html
```

Comment:
Is this sub common enough to put in the `rst_epilogue` in `conf.py`? It's appeared three times just in this PR. (I'd like to put common subs in a text file and actually read that in `conf.py` at some point)

Reply:
Those three are all the occurances of this URL in the repo. However, `rst_epilog` currently only contains replacements and not target definitions. I believe there is no real reason to not add link target definitions to `rst_epilog`, but maybe we should do that in a separate pr and add a whole bunch of common URLs (instead of just the webbrowser one).

Reply:
for sure, separate PR seems good, especially since I'd like to move it out to a separate file  👍